device/celestica/x86_64-cel_seastone-r0/sonic_platform/thermal_actions.py

```diff
@@ -0,0 +1,79 @@
+
+from sonic_platform_base.sonic_thermal_control.thermal_action_base import ThermalPolicyActionBase
+from sonic_platform_base.sonic_thermal_control.thermal_json_object import thermal_json_object
+from .thermal_infos import ChassisInfo
+from .helper import APIHelper
+
+
+@thermal_json_object('thermal_control.control')
+class ControlThermalAlgoAction(ThermalPolicyActionBase):
+    """
+    Action to control the thermal control algorithm
+    """
+    # JSON field definition
+    JSON_FIELD_STATUS = 'status'
+
+    def __init__(self):
+        self.status = True
+
+    def load_from_json(self, json_obj):
+        """
+        Construct ControlThermalAlgoAction via JSON. JSON example:
+            {
+                "type": "thermal_control.control"
+                "status": "true"
+            }
+        :param json_obj: A JSON object representing a ControlThermalAlgoAction action.
+        :return:
+        """
+        if ControlThermalAlgoAction.JSON_FIELD_STATUS in json_obj:
+            status_str = json_obj[ControlThermalAlgoAction.JSON_FIELD_STATUS].lower(
+            )
```

Comment:
Line break is unnecessary. Suggest moving closing parenthesis to line 30

Reply:
updated https://github.com/Azure/sonic-buildimage/pull/6089/commits/63ddb29bf99c6b547bc6c06d892757da2aa88451